templates/ecommerce/src/app/next/preview/route.ts

```diff
@@ -3,6 +3,11 @@ import { redirect } from 'next/navigation'
 
 import { payloadToken } from '../../_api/token'
 
+/**
```

Comment:
I don't think we need a note here. Only put this note in the READMEs and in the custom route handlers of the custom-server example